admin-action/package.json.liquid

```diff
@@ -6,7 +6,7 @@
 {%- if flavor contains "preact" %}
   "dependencies": {
     "preact": "^10.10.x",
-    "@shopify/ui-extensions": "2025-07-rc"
+    "@shopify/ui-extensions": "2025-10"
```

Comment:
Do we want to get away from using `2025.10.x` in our templates in favor of these tags? I guess they accomplish the same thing right?